src/atoms/TextInput/TextInput.web.js

```diff
@@ -1,3 +1,486 @@
+                                    name={name}
+                                    type={type}
+                                    placeholder={placeholder}
+                                    placeholderTextColor={placeholderTextColor}
+                                    onFocus={onFocus}
+                                    onBlur={onBlurText}
+                                    onChange={onChangeText}
+                                    hasText={hasText}
+                                    onSelect={onSelectText}
+                                    readonly={disabled}
+                                    disabled={disabled}
+                                    variant={variant}
+                                    hasPrefix={hasPrefix}
+                                    hasLeftIcon={hasLeftIcon}
+                                    maxLength={maxLength}
+                                    value={input}
+                                    ref={inputRef}
+                                    onKeyPress={onKeyPress}
+                                    {...automation(testID)}
+                                  />
+                                </Size>
+                              </Space>
+                            </Flex>
+                            {hasSuffix ? (
+                              <AccessoryText position="right" variant={variant} disabled={disabled}>
+                                {suffix}
+                              </AccessoryText>
+                            ) : null}
+                            {hasRightIcon ? (
+                              <AccessoryIcon
+                                variant={variant}
+                                name={iconRight}
+                                disabled={disabled}
+                                hasError={hasError}
+                                position="right"
+                              />
+                            ) : null}
+                          </InputContainer>
+                        </Size>
+                      </Flex>
+                      <Line isFocused={isFocused} hasError={hasError} disabled={disabled} />
+                    </FillContainer>
+                  </Space>
+                </Size>
+                {/* Bottom texts */}
+                {hasError || helpText || successText ? (
+                  <Flex flexDirection="row" justifyContent="space-between">
+                    <View>
+                      <Text
+                        errorText={errorText}
+                        helpText={helpText}
+                        successText={successText}
+                        disabled={disabled}
+                      />
+                      {maxLength !== undefined ? (
+                        <CharacterCount
+                          disabled={disabled}
+                          maxLength={maxLength}
+                          currentLength={isDefined(input) ? input.length : 0}
```

Comment:
```suggestion
                          currentLength={input.length || 0}
```

Reply:
Actually, you can just do `currentLength={input.length}` because default prop value is already `0`.